settings.gradle

```diff
@@ -42,6 +42,10 @@ include ":opentelemetry-all",
         ":opentelemetry-sdk-contrib-jaeger-remote-sampler",
         ":opentelemetry-bom"
 
+if(JavaVersion.current().isJava11Compatible()) {
```

Comment:
Have you considered testing against any Java 8 distributions that include JFR?

Reply:
I have not, once that becomes readily available we can look at building for JDK 8 as well. First step I think is to focus on JDK 11 and make that work well.